src/main/java/utils/DayOfWeeks.java

```diff
@@ -0,0 +1,13 @@
+package utils;
+
+import java.time.DayOfWeek;
+
+public class DayOfWeeks {
```

Comment:
이런 케이스는 네이밍이 겹치면 오해의 여지가 생길 수도 있기 때문에, 클래스 네이밍을 확실하게 구분해주는게 조금 더 좋다고 생각해요. 

```suggestion
public class DayOfWeekUtils {
```

Reply:
저는 Objects, Collections, Arrays와 같이 유틸성 클래스에 붙이는 관용적인 표현이라 이렇게 명명했는데 오히려 많이 사용하지 않는 관습인가 보네요!
반영하겠습니다! :)